types/ember-table/index.d.ts

```diff
@@ -1,105 +1,2 @@
 declare module 'ember-table/components/ember-table/component';
 declare module 'ember-table/components/ember-tbody/component';
-declare module '@gavant/ember-table/-private/sticky/legacy-sticky-polyfill';
-declare module '@gavant/ember-table/-private/sticky/table-sticky-polyfill';
```

Comment:
these 2 module declarations still need to be there as they are JS files copied over from the ember-table addon